app/models/user_track.rb

```diff
@@ -51,7 +70,7 @@ class DetermineAvailableExercisesIds
 
     # Get two datasets.
     # The first is the exercises with their counts of MATCHING preqreq concepts
-    # The second is the execises with the count of all their preqreq concepts
+    # The second is the exercises with the count of all their preqreq concepts
```

Comment:
```suggestion
    # The second is the exercises with the count of all their prereq concepts
```

Reply:
Done in a separate commit on master. Thanks 👍